filebeat/harvester/log.go

```diff
@@ -44,6 +44,29 @@ func (h *Harvester) Harvest() {
 		return
 	}
 
+	go func() {
+		select {
+		case <-h.prospectorDone:
+			close(h.done)
+		case <-h.done:
+			return
+		}
+
+	}()
+
+	if h.config.CloseTimeout > 0 {
```

Comment:
As far as I understand you, the current implementation works as I intended, but you would prefer to disable this in ucfg. The question then is what should be use in this case here? or the nonzero requirement should be removed?